src/common/tracing/src/minitrace.rs

```diff
@@ -108,13 +109,13 @@ pub fn init_logging(name: &str, cfg: &Config) -> Vec<Box<dyn Drop + Send + Sync
 
     // File logger
     if cfg.file.on {
-        let (normal_log_file, flush_guard) =
-            tracing_appender::non_blocking(RollingFileAppender::new(
-                Rotation::HOURLY,
-                &cfg.file.dir,
-                format!("databend-query-{name}"),
-            ));
-        guards.push(Box::new(flush_guard));
+        let rolling = RollingFileAppender::new(
+            Rotation::HOURLY,
+            &cfg.file.dir,
+            format!("databend-query-{name}"),
+        );
+
+        let normal_log_file = io::BufWriter::with_capacity(64 * 1024 * 1024, rolling);
```

Comment:
By the way, without flush guard, we could lost recent logs if application termination happended (at most 64MiB).

Reply:
Maybe it's better to wrap non_blocking appender with `BufWriter`.

Reply:
Or we can increase the `buffered_lines_limit` from 128 Ki to larger, maybe 64 Mi (like we did in this PR)

Reply:
`BufWriter` itself is a guard and will flush the buf in blocking mode when being dropped, I think that would be good enough?
```rust
#[stable(feature = "rust1", since = "1.0.0")]
impl<W: Write> Drop for BufWriter<W> {
    fn drop(&mut self) {
        if !self.panicked {
            // dtors should not panic, so we ignore a failed flush
            let _r = self.flush_buf();
        }
    }
}
```

Reply:
<img width="748" alt="image" src="https://github.com/datafuselabs/databend/assets/9637710/9afb2860-31c4-4f87-bf36-71f448e4d849">
When the buffer capacity is reached, `BufWriter` will perform a blocking flush.

Reply:
> `BufWriter` itself is a guard and will flush the buf in blocking mode when being dropped, I think that would be good enough?

Oh, I don't know about this. Lesson learnt.